firestore/firestore_snippets/listen_stop.go

```diff
@@ -1,44 +0,0 @@
-// Copyright 2020 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package main
-
-// [START fs_detach_listener]
-import (
-	"context"
-	"fmt"
-	"io"
-
-	"cloud.google.com/go/firestore"
-)
-
-// listenStop demonstrates how to detach a listener.
-func listenStop(w io.Writer, projectID string) error {
-	// projectID := "project-id"
-	ctx := context.Background()
-
-	client, err := firestore.NewClient(ctx, projectID)
-	if err != nil {
-		return fmt.Errorf("firestore.NewClient: %v", err)
-	}
-	defer client.Close()
-
-	qsnap := client.Collection("cities").Where("state", "==", "CA").Snapshots(ctx)
-	// Stop listening for changes.
-	qsnap.Stop()
-
-	return nil
-}
-
-// [END fs_detach_listener]
```

Comment:
I think it might make sense to keep this just for completeness; it would be odd if there are Go snippets for everything else on the page except for this.

Reply:
"Stop" doesn't make sense as a standalone example, though. You can't listen without stopping. So, we can include something. But, it seems like a square peg in a round hole.

Reply:
I added a comment in listenDocument sample for now

Reply:
The intent with this snippet is to have a stand alone sample showing users how they can detach a registered listener. The idea behind it is that sometimes listeners will be put into the 'background' and at some later time will need to be detached. For example, starting a listener as part of a users session and detach when the session closes/times out. We may not need an entire file to demonstrate this action, but we should ideally keep these region tags (possibly as part of another sample) so that they can be displayed in the docs here https://firebase.google.com/docs/firestore/query-data/listen#detach_a_listener .

Reply:
+1 to add the region tag to the existing file. When we embed the detach snippet, we can highlight the particular "detach" line(s).

Reply:
Done, put tags in the listenDocument